server/src/main/java/io/crate/types/DataType.java

```diff
@@ -107,7 +107,7 @@ public boolean precedes(DataType other) {
      * @param other the DataType to check conversion to
      * @return true or false
      */
-    public boolean isConvertableTo(DataType<?> other) {
+    public boolean isConvertableTo(DataType<?> other, boolean explicitCast) {
```

Comment:
`explicitCast` is not used at all in isConvertableTo

Reply:
It is: https://github.com/crate/crate/pull/9985/files#diff-932a1ef471972a95b5dfde013523b515R186

Reply:
Or here: https://github.com/crate/crate/pull/9985/files#diff-d9273c5d03d846ae406a74ae5af32ad5R109

Reply:
ah i see, tnx